scooby/report.py

```diff
@@ -217,9 +217,11 @@ def __repr__(self):
             text += '  '+txt+'\n'
         text += '\n'
 
+        # Get length of longest package
+        name_width = max(18, len(max(self._packages.keys(), key=len)))
```

Comment:
I would move this before "Platform/OS details", and replace the two instances of `{:>18}` there as well with `name_width`.

Awesome Bane, just learned something new: `max(..., key)`!

Reply:
On it, should we straight away restrict it assuming that otherwise at some point we will have the opposite problem with a toooooo long package name?
```
min(40, max(18, len(max(self._packages.keys(), key=len))))
```
40 would be half a regular terminal width, sounds like a good value.

Reply:
That sounds good to me! I will make the change